charts/spire/charts/spire-agent/values.yaml

```diff
@@ -227,8 +227,8 @@ workloadAttestors:
         fileName: ""
     ## @param workloadAttestors.k8s.disableContainerSelectors Set to true if using holdApplicationUntilProxyStarts in Istio
     disableContainerSelectors: false
-    ## @param workloadAttestors.k8s.useNewContainerLocator If true, enables the new container locator algorithm that has support for cgroups v2. Defaults to false
-    useNewContainerLocator: false
+    ## @param workloadAttestors.k8s.useNewContainerLocator If true, enables the new container locator algorithm that has support for cgroups v2. Defaults to true
+    useNewContainerLocator: true
```

Comment:
Should we note this change in the upgrade notes?

Reply:
its there already

Reply:
I dont see it in there. Just a simple `workloadAttestors.k8s.useNewContainerLocator is now enabled by default.` should do it.

Reply:
https://github.com/spiffe/helm-charts-hardened/blob/spire-1.11/charts/spire/README.md?plain=1#L95